reason-react-native/src/apis/StyleSheet.md

```diff
@@ -1,20 +1,104 @@
 ---
 id: apis/StyleSheet
 title: StyleSheet
-wip: true
+officialDoc: https://facebook.github.io/react-native/docs/stylesheet
 ---
 
+`StyleSheet` mainly has a [method](#create) to create an object containing many
+styles, similar to a CSS StyleSheet. This is helpful to keep style definitions
+at fewer locations, away from the render function. Separating concerns in your
+code in this manner should help readability. Named styles might also help
+differentiate between your components. Finally, allowing components to refer to
+styles by ID, it also reduces the amount of data transferred over the JS bridge.
+
+There is also a [method](#flatten) to merge multiple styles into a single style
+and several [constants](#constants) for commonly used style definitions.
+
+## Methods
+
+### `create`
+
+Takes and returns `Js.t('a)` objects, with `string` keys and `Style.t` values.
+Individual styles can be accessed using `##`, followed by the name specified as
+the key.
+
+```reason
+create: Js.t('a) => Js.t('a)
+```
+
+### `flatten`
+
+Takes an array of styles (of type `Style.t`) and returns a single style (also of
+type `Style.t`).
```

Comment:
Maybe add a comment that using styles created with `Style.array` in `StyleSheet.create` will lead to a runtime error, so `StyleSheet.flatten` *must* be used in that case. (`Style.array` is fine for passing styles to component props though.)

Reply:
Added such comments